scripts/rollup/build.js

```diff
@@ -18,11 +18,11 @@ if (process.env.CIRCLE_NODE_TOTAL) {
   packages = packages.filter((_, i) => i % nodeTotal === nodeIndex);
 }
 
-(async () => {
-  for (const package of packages) {
-    const packageName = path.relative(workspaceRoot, package);
-    console.log('> Building', packageName);
+const builds = packages.map(package => {
+  const packageName = path.relative(workspaceRoot, package);
+  console.log('> Building', packageName);
 
+  return new Promise(async function(resolve, reject) {
```

Comment:
The `new Promise` wrapper here doesn't seem to be necessary

Reply:
Changed!